enterprise/models.py

```diff
@@ -763,9 +772,39 @@ def unenroll(self, course_run_id):
         return False
 
     def update_session(self, request):
-        """Update the session of a request for this learner."""
+        """
+        Update the session of a request for this learner.
+        """
         request.session['enterprise_customer'] = self.enterprise_customer.serialized
 
+    def create_order_for_enrollment(self, course_run_id):
+        """
+        Create an order on the Ecommerce side for tracking the course enrollment of a enterprise customer user.
+        """
+        LOGGER.info("Creating order for enterprise customer with id [%s] for enrollment in course with id: [%s]",
```

Comment:
Note on terminology:
- An "enterprise customer" is the actual business we're partnering with. 
- A "learner" is the edX-y way to say what most people would call a "user". You can also just use "user".
- An "enterprise customer user" is a learner that's attached to an enterprise customer. 

Also, confusingly, an `EnterpriseCustomerUser` has an `id` that's different from the `User`'s `id` (DBs and all that...).

So. This should probably say "...for enterprise learner with user id [%s] ...".